execution/setContext.sh

```diff
@@ -274,7 +280,7 @@ if [[ -n "${AZID}" ]]; then
         . ${AUTOMATION_DIR}/setCredentials.sh "${ACCOUNT}"
 
     fi
-    az account set --subscription "${AZID}" --output none || exit $?
+    az account set --subscription "${AZID}" --output none
```

Comment:
I've found that output type of none is annoying to debug with. I've been using [this method](https://github.com/hamlet-io/engine-plugin-azure/blob/4dd309eb9f8ff7596e6a7eb551d289f8de07dff4/azure/utility.sh#L113) in the utility script.

Reply:
Fixed, used the debug logging functions instead of checking the variable. You can have GENERATION_LOG_LEVEL = info as a set variable